packages/client/components/MeetingControlBar.tsx

```diff
@@ -200,7 +200,8 @@ const MeetingControlBar = (props: Props) => {
                 <StageTimerControl
                   {...tranProps}
                   cancelConfirm={cancelConfirm}
-                  defaultTimeLimit={DEFAULT_TIME_LIMIT[phaseType]}
+                  //default time limit of Poker is 3
+                  defaultTimeLimit={DEFAULT_TIME_LIMIT[phaseType] || 3}
```

Comment:
-1 is there a way that we can accomplish this without hardcoding the poker time limit?

Reply:
Thanks for the suggestion, working on it

Reply:
I am using Scope: 3 and Estimate: 5.
Is that ok?

Reply:
@nickoferrall 
Polite reminder